migrations/1_initial_migrations.js

```diff
@@ -0,0 +1,5 @@
+let Migrations = artifacts.require('./Migrations.sol');
+
+module.exports = function (deployer) {
+  deployer.deploy(Migrations);
```

Comment:
don't we need to deploy all contracts here one by one, or as a list or something?